Chrysalis Docs Review IV (#72)
iotaledger/chrysalis-docs: docs/guides/hub_guide.md

# IOTA Hub Migration Guide

> > This guide outlined how to switch from IOTA Hub to using wallet.rs or its bindings instead.

Reviewer: Unless this guide no longer outlines the how to switch, I'd suggest the following edit.

```suggestion
This guide outlines how to switch from IOTA Hub to using [Wallet.rs](https://wallet-lib.docs.iota.org/) or its [bindings](https://wallet-lib.docs.iota.org/libraries/index.html) instead.
```

Author: Yep!

> Because of the unique features of IOTA 1.0, it was difficult to manage IOTA transactions with just a library. With the Chrysalis update, the whole protocol was updated to be more accommodating to industry wide standards and developer friendliness. 

Reviewer: ```suggestion
Because of the unique features of IOTA 1.0, it was difficult to manage IOTA transactions with just a library. With the Chrysalis update, we have updated the whole protocol to be more accommodating to industry wide standards and developers.
```

> IOTA Hub was deprecated with the Chrysalis upgrade and does not work with the new protocol changes. If you somehow still use Hub, we ask you to utilize our new wallet library (iota.rs or bindings) where you can easily manage IOTA addresses, deposits, and withdrawals for user accounts.

Reviewer: ```suggestion 
IOTA Hub was deprecated with the Chrysalis upgrade and will not work with the new protocol changes. If you somehow still use Hub, we recommend that you to switch our new wallet library [Iota.rs](https://client-lib.docs.iota.org/), or its [bindings](https://client-lib.docs.iota.org/libraries/index.html), where you can easily manage IOTA addresses, deposits, and withdrawals for user accounts.
```

> To upgrade from Hub to a Chrysalis implementation you needed to:

Reviewer: ```suggestion
To upgrade from Hub to a Chrysalis implementation you will need to:
```

>  - Integrate the Chrysalis network using wallet.rs as mentioned in the Exchange section.

Reviewer: ```suggestion
 - Integrate the Chrysalis network using [Wallet.rs](https://wallet-lib.docs.iota.org/) as mentioned in the [Exchange section](./exchange_guide.md).
```

 - Pause withdrawals/deposits.
 - Make sure all balances have been swept and all deposits have been processed.

>  - Transfer all IOTA to a generated migration address (instructions to be provided).

Reviewer: We should probably provide the instructions here, or at least a link to the migration-mechanism file.

Author: These _are_ the instructions :(

>  - Once migrated, transfer the IOTA coins to your hot wallet account on your wallet.rs implementation.

Reviewer: ```suggestion
Once migrated, transfer the IOTA coins to your host wallet account on your wallet.rs implementation.
```
